proto/api/api.proto

```diff
@@ -105,8 +105,14 @@ message GetActionsByBlockRequest {
   uint64 count = 3;
 }
 
+message Action {
```

Comment:
it feels strange to access iotextypes.Action  like getActionsRespons.actions[n].action

I suggest naming it `MintedAction` or `ActionResult` ... or other kinds of action you can name it